packages/workspace-store/src/schemas/workspace-specification/index.ts

```diff
@@ -0,0 +1,26 @@
+import { compose } from '@/schemas/v3.1/compose'
+import { OpenAPIDocumentSchema } from '@/schemas/v3.1/strict/openapi-document'
+import { WorkspaceMetaSchema } from '@/schemas/workspace'
+import { ConfigSchema } from '@/schemas/workspace-specification/config'
+import { InfoSchema } from '@/schemas/workspace-specification/info'
+import { Type, type Static } from '@sinclair/typebox'
+
+export const WorkspaceSpecificationSchema = compose(
+  Type.Object({
+    workspace: Type.Union([Type.Literal('1.0.0')]),
```

Comment:
```suggestion
    workspace: Type.Union([Type.Literal('draft')]),
```

maybe we start with `draft` as a version? just a cosmetic change